### PR TITLE
adding temporary categorize_dataset function

### PR DESCRIPTION
**Pull request by laijasmine**

### CODE REVIEW

File: R/categorize_dataset.R [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#'
+#' @return NULL the result wil be written to an external google sheet
+
+#' @examples
+#' # categorize_dataset("doi:10.18739/A2QJ77Z09", c("biology", "oceanography"), "your name")
+#' @author Jasmine Lai
+#' @export
+categorize_dataset <- function(doi, themes, coder, test = F) {
+  stopifnot(length(themes) > 0 & length(themes) < 5)
+  stopifnot(grepl(doi, "doi"))
```
Comment (jeanetteclark) [inline R/categorize_dataset.R:19]:
```suggestion
  stopifnot(grepl("doi", doi))
```

syntax error here I think

File: R/categorize_dataset.R [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  #Select test sheet
+  if (test) {
+    ss <- "https://docs.google.com/spreadsheets/d/1GEj9THJdh22KCe1RywewbruUiiVkfZkFH8FO1ib9ysM/edit#gid=1479370118"
+  } else {
+    ss <- "https://docs.google.com/spreadsheets/d/1S_7iW0UBZLZoJBrHXTW5fbHH-NOuOb6xLghraPA4Kf4/edit#gid=1479370118" # offical
+  }
+
+  #checking for valid themes
+  accepted_themes <- googlesheets4::read_sheet(ss, sheet = "categories", col_names = F)
```
Comment (jeanetteclark) [inline R/categorize_dataset.R:32]: Maybe we consider wrapping this in a `supressMessages`? I don't love the amount of output it spits out

**Comment by laijasmine** [comment]
> You might also want to consider checking to see if the DOI or a previous versions already exists in the sheet and if it does, handle it somehow (error, overwrite that row)

I think that will take me a bit longer it wasn't as quick as I thought so I might just merge these changes first then work on that after.. I don't suspect it to be too much of a problem in the short term but will be needed soon

**Comment by jeanetteclark** [comment]
Yeah I agree - lets merge this version and then work on figuring out how to handle the repeats in the next iteration. We should do it relatively soon though I think.

I think the various scenarios, and possible solutions, are:

- that identifier and no previous versions are not in the sheet: add the row
- the identifier is already in the sheet: 
        + categories are the same: warning message, but do nothing
        + categories are different: error message, do not insert

**Pull request merged**